### ISSUE TITLE
Background section is a mess

### ISSUE THREAD

**Issue by gustafl** [w3c/selection-api#84]
The [background section](http://w3c.github.io/selection-api/#background) of the current editors draft does not follow the language of standards, or the tradition of W3C recommendations. One can easily point to a number of breaches of style:

- Writing in first person ("So I changed the spec...").
- Referring to specific browsers (even _specific versions_). It's called "user agent" for a reason.
- Referring to specific forum discussions (using a broken link, by the way).
- Going into details in a what is supposed to be a background section.

In short, the current text reads like a blog post and has no place in a standards document. It should be removed or completely rewritten.

**Comment by hugoholgersson** [comment]
Would you be able to send a pull request @gustafl? I could help out by reviewing it.

**Comment by rniwa** [comment]
Yeah, please feel free to send a PR.
